int32 msg_id
bool remote_request
int8 DLC
uint8[] data
int32 CRC_seq
bool ACK
